packages/demand-deeplink/src/parse.ts

```diff
@@ -177,9 +177,16 @@ function parseSearchString(query: string) {
   const data = new URLSearchParams(query)
 
   data.forEach((v, k) => {
-    const value = v && v.trim()
+    let value = v && v.trim()
     const key = k && k.trim().toLowerCase()
 
+    if (key.includes('time')) {
```

Comment:
I think it is too hacky.
What do you think about testing value against date time format (with space before timezone part)?
This way we can avoid any costom/future problems with date/time related params.

We also have to write test for this case (when timezone with `+` passed